docs/configure.md

```diff
@@ -72,6 +73,8 @@ core principles for runsettings.
    test run via the test platform APIs. It may use it to read its own settings
    or take decisions (e.g. an adapter needs to disable appdomains if
    `DisableAppDomain` setting is provided via run settings).
+5. Users can now configure settings used for legacy scenarios like deployment,
+   scripts using run settings.
```

Comment:
Clarify that legacy scenarios are only applicable for MSTest v1 framework.

Reply:
Also add that this enables users to use all the settings that were available in testsettings using runsettings.

Reply:
Covered these in the LegacySettings section below.